include/seqan3/alphabet/concept.hpp

```diff
@@ -259,14 +246,12 @@ namespace seqan3
  *
  * It acts as a wrapper and looks for three possible implementations (in this order):
  *
- *   1. A free function `to_char(your_type const a)` in the namespace of your
- *      type (or as `friend`). The function must be marked `noexcept` (`constexpr` is not required,
- *      but recommended) and the return type be of the respective char representation (usually a small integral type).
- *   2. A free function `to_char(your_type const a)` in
- *      `namespace seqan3::custom`. The same restrictions apply as above.
+ *   1. A free function `to_char(your_type const a)` in the namespace of your type (or as `friend`).
+ *   2. A free function `to_char(your_type const a)` in `namespace seqan3::custom`.
  *   3. A member function called `to_char()`.
- *      It must be marked `noexcept` (`constexpr` is not required, but recommended) and the return type be
- *      of the respective char representation.
+ *
+ * Functions are only considered for one of the above cases if they are marked `noexcept` (`constexpr` is not required,
+ * but recommended).
```

Comment:
No restriction for the return type here?

Reply:
No, currently not. We are still discussing whether we require convertibility to `char`...

Reply:
I vote for doing it! But you can discuss/implement that later – I'm approving for now.